{
  "name": "babel4catch",
  "version": "1.0.10",
  "description": "babel plugin for funtion to use try/catch",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [
    "babel-plugin",
    "try",
    "catch"
  ],
  "author": "xsf",
  "license": "ISC",
  "dependencies": {
    "@babel/core": "^7.11.6",
    "@babel/parser": "^7.11.5"
  }
}
